<launch>

  <node pkg="nodelet" type="nodelet" name="managerWall" args="manager" output="screen">	
    <param name="num_worker_threads" value="4" />
  </node>


  <node pkg="nodelet" type="nodelet" name="wall_detector" args="load romina2/WallDetectorNodelet managerWall" output="screen">
    <param name="imgThrs" value="150"/>
    <param name="lineVoteThrs" value="25"/>
    <param name="lineMinLen" value="10"/>
    <param name="lineMaxGap" value="30"/>
    <remap to="/cam/camera_info" from="~camera_info" />
    <remap to="/cam/image_raw" from="~image" />
  </node>


  
</launch>
